resources/config-defaults.edn

```diff
@@ -22,6 +22,9 @@
  ;; Supported authentication methods: :shibboleth, :fake-shibboleth, :oidc
  :authentication :shibboleth
 
+ ;; Log additional details about authentication
+ :debug-authentication false
```

Comment:
I would rather have this named `:log-authentication-details` as some instance may want to do it anyway in production.

Reply:
I'll change it